crates/sui-sdk-types/address.rs

```diff
@@ -0,0 +1,263 @@
+#[derive(Clone, Copy, Hash, PartialEq, Eq, PartialOrd, Ord)]
```

Comment:
this file seems like it was added by mistake?

Given how long this PR is, it may be beneficial to squash all the commits yourself to make it a bit easier to work with

Reply:
Very likely, I will do a check given that I rebased on #60, maybe I missed something there.